parachains/runtimes/assets/test-utils/src/test_cases_over_bridge.rs

```diff
@@ -0,0 +1,447 @@
+			let _ = <pallet_balances::Pallet<Runtime>>::deposit_creating(
+				&alice_account,
+				alice_account_init_balance,
+			);
+			// SA of target location needs to have at least ED, anyway making reserve fails
+			let _ = <pallet_balances::Pallet<Runtime>>::deposit_creating(
+				&reserve_account,
+				existential_deposit,
+			);
+
+			// we just check here, that user remains enough balances after withdraw
+			// and also we check if `balance_to_transfer` is more than `existential_deposit`,
+			assert!(
+				(<pallet_balances::Pallet<Runtime>>::free_balance(&alice_account) -
+					balance_to_transfer.into()) >=
+					existential_deposit
+			);
+			// SA has just ED
+			assert_eq!(
+				<pallet_balances::Pallet<Runtime>>::free_balance(&reserve_account),
+				existential_deposit
+			);
+
+			// local native asset (pallet_balances)
+			let asset_to_transfer = MultiAsset {
+				fun: Fungible(balance_to_transfer.into()),
+				id: Concrete(native_asset),
+			};
+
+			// destination is (some) account relative to the destination different consensus
+			let target_destination_account = MultiLocation {
+				parents: 0,
+				interior: X1(AccountId32 {
+					network: Some(bridged_network),
+					id: sp_runtime::AccountId32::new([3; 32]).into(),
+				}),
+			};
+
+			// do pallet_xcm call reserve transfer
+			assert_ok!(<pallet_xcm::Pallet<Runtime>>::limited_reserve_transfer_assets(
+				RuntimeHelper::<Runtime>::origin_of(alice_account.clone()),
+				Box::new(target_location_from_different_consensus.into_versioned()),
+				Box::new(target_destination_account.into_versioned()),
+				Box::new(VersionedMultiAssets::from(MultiAssets::from(asset_to_transfer))),
+				0,
+				weight_limit,
+			));
+
+			// check alice account decreased about all balance_to_transfer
+			assert_eq!(
+				<pallet_balances::Pallet<Runtime>>::free_balance(&alice_account),
+				alice_account_init_balance - balance_to_transfer.into()
+			);
+
+			// check reserve account
+			// check reserve account increased about all balance_to_transfer
+			assert_eq!(
+				<pallet_balances::Pallet<Runtime>>::free_balance(&reserve_account),
+				existential_deposit + balance_to_transfer.into()
+			);
+
+			// check events
+			// check pallet_xcm attempted
+			RuntimeHelper::<Runtime>::assert_pallet_xcm_event_outcome(
+				&unwrap_pallet_xcm_event,
+				|outcome| {
+					assert_ok!(outcome.ensure_complete());
+				},
+			);
+
+			// check that xcm was sent
+			let xcm_sent_message_hash = <frame_system::Pallet<Runtime>>::events()
+				.into_iter()
+				.filter_map(|e| unwrap_xcmp_queue_event(e.event.encode()))
+				.find_map(|e| match e {
+					cumulus_pallet_xcmp_queue::Event::XcmpMessageSent { message_hash } =>
+						Some(message_hash),
+					_ => None,
+				});
+
+			// read xcm
+			let xcm_sent =
+				RuntimeHelper::<HrmpChannelSource>::take_xcm(local_bridge_hub_para_id.into())
+					.unwrap();
+			println!("xcm_sent: {:?}", xcm_sent);
```

Comment:
remove println!?

Reply:
thx, removed and added additional assert with simple XCM matcher